extensions/tekton/examples/src/main/java/io/fabric8/tekton/api/examples/CancelTask.java

```diff
@@ -0,0 +1,31 @@
+package io.fabric8.tekton.api.examples;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.stereotype.Component;
+import io.fabric8.knative.internal.pkg.apis.Condition;
+import io.fabric8.tekton.client.DefaultTektonClient;
+import io.fabric8.tekton.client.TektonClient;
+import io.fabric8.tekton.pipeline.v1beta1.TaskRun;
+
+public class CancelTaskRun {
+
+  public static void main(String[] args) {
+    try (TektonClient tektonClient = new DefaultTektonClient()) {
+      
+      TaskRun taskRun = tektonClient.v1beta1().taskRuns().list().getItems().get(0);
```

Comment:
As far as I remember TaskRun is a Namespaced resource, Do you think we should specify `inNamespace("your-ns")`?

Reply:
Hi @rohanKanojia not a problem. I was using what I had configured via `oc project`. I updated with what was in TaskRunCreate